script/example.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+import yaml
+import json
+from argparse import ArgumentParser
+
+def action_json(args):
+
+    with open('scripts/example_data.json') as json_file:
+        print(json.dumps(json.loads(json_file.read()), indent=2))
+
+def action_yaml(args):
+    with open('scripts/example_data.yaml') as yaml_file:
```

Comment:
For consistency and readability, use a context manager (`with`) with `yaml.safe_load()` directly instead of reading the file into memory entirely before parsing.
```suggestion
with open('scripts/example_data.yaml', 'r') as yaml_file:
    print(yaml.dump(yaml.safe_load(yaml_file), indent=2))
```